src/widgets/WidgetApi.ts

```diff
@@ -23,6 +23,7 @@ export enum Capability {
     Screenshot = "m.capability.screenshot",
     Sticker = "m.sticker",
     AlwaysOnScreen = "m.always_on_screen",
+    ReceiveTerminate = "m.receive_terminate",
```

Comment:
We can't use the `m.*` namespace for this - please use `im.vector.receive_terminate` for now.